tests/unit/api/bucketHead.js

```diff
@@ -37,15 +37,4 @@ describe('bucketHead API', () => {
             });
         });
     });
-
-    it('should return a success message if ' +
```

Comment:
it asserts result, a string I removed as a param for the callback in routes, and replaced with corsHeaders. I think the string was included as a param specifically for this test?
Anyway I thought for some reason there was another test that covered success but I don't see it; so I can replace this test with one that asserts no error if that's sufficient.